rnc2rng/parser.py

```diff
@@ -70,6 +70,12 @@ def __repr__(self):
     'SEQ', 'SOME', 'TEXT',
 ]
 
+for _name in NODE_TYPES:
+    assert _name not in globals()
+    globals()[_name] = _name
+
+del _name
```

Comment:
`del` is optional since this variable is private.  But I like to do it, as people are curious and they look at what is hidden inside the module that they might be able to use.  In this case, there is no possible use for the symbol.

I went with `_name` previously, as that name suggests it is a string.  `nt` tells me nothing, and took me a few seconds to realise what it was.  I've used `_node_type`, as that at least gives it a name which is private and self-explanatory.